daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1946,3 +1946,25 @@ class RazerBlade182023(_RippleKeyboard):
                'set_ripple_effect', 'set_ripple_effect_random_colour']
 
     DEVICE_IMAGE = "https://dl.razerzone.com/src2/9676/9676-1-en-v1.png"
+
+
+class RazerProTypeUltraWired(_RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Pro Type Ultra (Wired)
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Pro_Type_Ultra(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0277
+    METHODS = ['get_device_type_keyboard', 'set_static_effect', 'set_breath_single_effect', 'get_battery', 'is_charging', 'get_keyboard_layout']
```

Comment:
The device really doesn't support more effects than this? At least `set_none_effect` maybe to turn off the lighting? Or brightness controls?

Reply:
It has brightness controls, both classes inherit `RazerDeviceBrightnessSuspend` so there's no need to declare those methods in the `METHODS` property.
Here are a few screenshots of that device from Razer Synapse on Windows to view the exact functionality:

![Screenshot 2023-09-25 230124](https://github.com/openrazer/openrazer/assets/86299799/948b4618-543f-4b6b-aacf-199976690ee9)

![Screenshot 2023-09-25 230146](https://github.com/openrazer/openrazer/assets/86299799/e6899255-6a18-437f-bffd-d87f8014167c)

![Screenshot 2023-09-25 230231](https://github.com/openrazer/openrazer/assets/86299799/96546f27-5efc-478c-91b7-d288763f796a)